src/js/components/covid19/assistanceListing/SpendingByCFDA.jsx

```diff
@@ -61,6 +61,13 @@ const SpendingByCFDA = () => {
         Analytics.event({ category: 'COVID-19 - Award Spending by CFDA', action: `${activeTab} - click` });
     };
 
+    // Keep track of previous tab to prevent duplicate requests in summary
+    const prevTabRef = useRef();
+    useEffect(() => {
+        prevTabRef.current = activeTab;
```

Comment:
This duplicates our state. A `ref` [should be used sparingly for `mutuable` data](https://reactjs.org/docs/hooks-reference.html#useref). Typically this means a DOM Node. This is how we use it. In this case, we're putting immutable data that duplicates a property already in state inside a ref. We should just use the state property here.

Reply:
This particular use case I think makes sense to not use state sense I am trying to track the previous value of the state: https://reactjs.org/docs/hooks-faq.html#how-to-get-the-previous-props-or-state

To use state would mean adding an additional update in the code for each state update when you can let this ref handle tracking that.

Reply:
Oh my! I had no idea this was a convention recommended in the documentation. Nicely done sir.

Reply:
I will say if this becomes a more common thing then we should create a custom hook, but right now this is just a one off use case.